dialogue-core/src/main/metrics/dialogue-core-metrics.yml

```diff
@@ -16,6 +16,10 @@ namespaces:
           Number of requests that are actively running. The `stage` may refer to `running` requests actively
           executing over the wire or `processing` which may be awaiting a client or backing off for a retry.
           Note that running requests are also counted as processing.
+      requests.queued:
+        type: gauge
```

Comment:
Can we use a counter, and wrap both the atomic int + counter with a simple abstraction to expose `incrementAndGet` `decrementAndGet` and `get`? This way we don't have to worry about gauge weirdness.

Reply:
any reason to not just use the counter as a source of truth instead of atomic int?